product_configurator/wizard/product_configurator.py

```diff
@@ -179,11 +175,6 @@ def get_onchange_domains(
             check_avail_ids = list(
                 set(check_avail_ids) - (set(line.value_ids.ids) - set(avail_ids))
             )
-            # Include custom value in the domain if attr line permits it
-            if line.custom and custom_val.id in avail_ids:
-                domains[field_name][0][2].append(custom_val.id)
-                if line.multi and vals and custom_val.id in vals[0][2]:
-                    continue
```

Comment:
It has been removed because it has already been taken care of and works without this code.
at line 174 it is adding `domains[field_name] = [("id", "in", avail_ids)]`
and checking now that  `custom_val.id in avail_ids` then add again `domains[field_name][0][2].append(custom_val.id)` which is really not needed IMO.

Please let me know if I am missing/overlooking the code.

Reply:
I see, I think you're right, indeed.
Thanks!